front/app/services/permissions/rules/routePermissions.ts

```diff
@@ -11,41 +11,37 @@ import {
 import { IUser } from 'services/users';
 import { IAppConfigurationData } from 'services/appConfiguration';
 
-export const MODERATOR_ROUTES = [
+const MODERATOR_ROUTES = [
   '/admin', // currently redirects to /admin/dashboard
+  '/admin/dashboard',
   '/admin/projects',
   '/admin/messaging',
   '/admin/ideas',
   '/admin/ideas/import',
   '/admin/workshops',
   '/admin/processing',
-  '/admin/dashboard',
   '/admin/moderation',
   '/admin/insights',
   '/admin/content-builder',
 ];
 
 export const isModeratorRoute = (item: IRouteItem) => {
   return MODERATOR_ROUTES.some((moderatorRoute) => {
-    return item.path.includes(moderatorRoute);
+    return item.path === moderatorRoute;
```

Comment:
What if it's a nested url? E.g., `messaging/texting`. Maybe better to remove locale and use `startsWith`? 

https://github.com/CitizenLabDotCo/citizenlab/pull/2248/files#diff-390a6d9b051298c41815e3d5fd1e636ddd197cd0daca0f30a8e163d41c4c95efR474

Reply:
Yes indeed, good point! That's why I was still going to ask for a review. :D

I think we should also write tests.